### PR TITLE
Add logic

### PR DESCRIPTION
**Pull request by Edward-Kabue**
## Did the following:

- Created a collection that keeps a list of books.
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection
- Display all books saved in the collection in the top part of the page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Microverse capstone
+## Awesome books
```
Comment (TedLivist) [inline README.md]: - [ ] In your README, kindly add instructions on how to set up and run the app locally.

File: README.md [hunk 2]
```diff
@@ -11,11 +11,6 @@ Developed a project which displays a list of books. The user can add books by pr
...
 - Javascript
 
-## Acknowledgements
-
-- Original design idea by [Cindy Shin in Behance.](https://www.behance.net/adagio07)
-- [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)
-
 ## Authors
 
 - GitHub: [@Edward-Kabue](https://github.com/Edward-Kabue)
```
Comment (TedLivist) [inline README.md]: - [ ] Ensure there are commits from the other author.

- [ ] This is a collaborative project, and you are missing the other author in your documentation, kindly add the author details here.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  static removeBook(title) {
+    const books = Book.getBooks();
+    const otherBook = books.filter((item) => item.title !== title);
+
+    localStorage.setItem("books", JSON.stringify(otherBook));
+  }
+}
+
+document.addEventListener("DOMContentLoaded", Book.displayBooks);
```
Comment (TedLivist) [inline app.js:69]: - [ ] On page load, your app does not load the books from the `localStorage`. When I add books they are stored in the `localStorage` but when I reload the page, the books are not displayed. Kindly fix that.

  Books are added | Page is reloaded
  :---:|:---:
  matchoudi-1 | matchoudi-2

File: app.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  Book.clearField();
+
+  Book.addBook(book);
+}
+
+document.getElementById("form-list").addEventListener("click", remove);
+function remove(e) {
+  Book.deleteBook(e.target);
+  Book.removeBook(e.target.parentElement.previousElementSibling.textContent);
+}
```
Comment (TedLivist) [inline app.js]: - [ ] Your remove book functionality does not work as expected. It removes the book from the page but not from the localStorage. Kindly ensure that books are also removed from the localStorage.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Edward-Kabue,

Good job so far!
✔️ Linters are set up and working correctly
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 4 above)
Comment (Amrhub) [inline app.js]: - [ ] As requested by the previous code reviewer, Let's make sure when we delete a book it's also removed from the localStorage

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Edward-Kabue, 👋🏻 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight ✔️ 
Excellent communication with previous code reviewer 😇 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Meltrust** [comment]
Hello @LionRouge1, @Edward-Kabue.  

I'm your new reviewer, and I can see the bug persists, the books are not being deleted from local storage.

I can help you debug this.   I already contacted you on Slack.  I'll wait on Slack for 20 minutes, if I get no response, I will be forced to request changes.

Please, kindly respond.

**Review by Meltrust — APPROVED** [review]
Hi @Edward-Kabue,


## Wow, you did it 🎉
Brilliant
## Thank you for correcting your readme, and for correcting the bug that prevented deletion of books from localStorage, as requested by the previous reviewer 💪 🥇 ㊗️ 
## And thank you for responding on Slack!
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**